script/source-setup/npm

```diff
@@ -15,3 +15,9 @@ if [ "$1" == "-f" ]; then
 fi
 
 npm install
+
+# setup test fixtures with missing dependencies
+mkdir -p missing-dependencies
```

Comment:
🤔 I'm not sure whether there should be an entirely new folder of fixtures for the two added test cases.  How do you feel about doing something similar to [these go tests](https://github.com/github/licensed/blob/e223ccab5f8299a278d4679c5c8c6e0982ce545e/test/sources/go_test.rb#L128-L151) or [this bundler tests](https://github.com/github/licensed/blob/e223ccab5f8299a278d4679c5c8c6e0982ce545e/test/sources/bundler_test.rb#L198-L209)?

Reply:
Ok, followed bundler example (as it looks slightly cleaner to me).